### PR TITLE
Validate config.json and federations against the schemas

### PR DESCRIPTION
**Pull request by nl0**
### TODO

- [x] remove schemas from `lambdas/navigator_config` and adjust python code to use `catalog/*-schema.json` @nl0 Done in #941

### CODE REVIEW

**Comment by nl0** [comment]
@dimaryaz @akarve should we move federation schema to the catalog as well and use it for validating federations? i'm asking bc i remember talking about ditching federation jsons or smth.

also @dimaryaz could you please adjust navigator_config code to use `catalog/config-schema.json` and remove the original file?

**Comment by nl0** [comment]
i've implemented validating federations

**Comment by akarve** [comment]
@dimaryaz  - this PR settles where json schemas should live. I think we can now eliminate `lambdas/navigator_config/index.py.validate_configs`, update `navigator_config/setup.py` (which references the json schemas), and finally update `deployment/.../env_schema.py` to point to the new locations on GitHub

File: catalog/app/utils/Config.js [hunk 1]
```diff
@@ -93,10 +60,19 @@ const fetchConfig = async (path) => {
   }
 }
 
+const AUTH_MAP = {
+  ENABLED: true,
+  DISABLED: false,
+  SIGN_IN_ONLY: 'SIGN_IN_ONLY',
+}
+
 const transformConfig = (cfg) => ({
```
Comment (dimaryaz) [inline catalog/app/utils/Config.js]: Haha, I didn't realize this function existed when I added `fixConfig`.

**Pull request merged**